md_template/model.md

```diff
@@ -0,0 +1,51 @@
+This is a simple template for setting up automatic report generation with GPkit.
+To generate this file, run
+```
+make model.pdf
+```
+in this directory.
+
+\section{Example Model}
+
+This example model minimizes $x + 1/x$ subject to the constraint
+$$x \ge x_{min}.$$
+The description above is hard coded as opposed to being derived from the Python
+code. This is a simple and effective way to document model equations.
+
+The full GP, derived from the Python implementation of the model, is
+```python
+#inPDF: replace with gp.generated.tex
+from simple_gp import SimpleGP
+m = SimpleGP()
+with open("gp.generated.tex", "w") as f:
+    f.write(m._repr_latex_())
```

Comment:
If you use `m._latex(excluded=["models"])` you can print out the latex without the model string.